src/main/java/seedu/duke/Main.java

```diff
@@ -44,10 +47,19 @@ private void run(String[] args) {
      * storage file, then showing the welcome message.
      */
     private void start() {
+        this.storage = new Storage();
+        //TODO initialize this with the storage file after merging
         this.exerciseItems = new ExerciseList();
-        this.foodItems = new FoodList();
-        this.profile = new Profile();
         this.ui = new Ui();
+        try {
+            this.profile = storage.loadProfileFile();
+            this.foodItems = storage.loadFoodListFile();
+        } catch (UnableToReadFileException e) {
```

Comment:
perhaps you might want to change the error into messages instead of printing the whole stacktrace for future revisions.

Reply:
Oops yeah forgot to remove it, but i'll leave it there for now for debugging purposes 👍 Thanks yi zhi